tests/unit/dynamic_dependency_test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Even better, I think this can simply be switched to ``/bin/sh``, I don't see any bash specific syntax in there.